yarn.lock

```diff
@@ -1930,39 +1914,14 @@ babel-preset-jest@^24.9.0:
     "@babel/plugin-syntax-object-rest-spread" "^7.0.0"
     babel-plugin-jest-hoist "^24.9.0"
 
-babel-runtime@^6.22.0, babel-runtime@^6.23.0, babel-runtime@^6.26.0:
+babel-runtime@^6.23.0:
   version "6.26.0"
   resolved "https://registry.yarnpkg.com/babel-runtime/-/babel-runtime-6.26.0.tgz#965c7058668e82b55d7bfe04ff2337bc8b5647fe"
   integrity sha1-llxwWGaOgrVde/4E/yM3vItWR/4=
   dependencies:
     core-js "^2.4.0"
```

Comment:
Should you update `yarn.lock` as well?  
AFAIK, `core-js` should not be a dependency anymore.  
Not sure how `yarn.lock` works, though.

Refer to https://github.com/ScriptedAlchemy/webpack-external-import/issues/47 for details

Reply:
Probably used by some devDependencies. NVM if that is the case.

Reply:
You're right, they are from devDeps